### PR TITLE
140 Add license description

### LINKED ISSUE #140: clarify the pre-trained models licensing
### Discussed in https://github.com/Project-MONAI/model-zoo/discussions/139

<div type='discussions-op-text'>

<sup>Originally posted by **dyollb** September 26, 2022</sup>
I was looking at the pre-trained model https://github.com/Project-MONAI/model-zoo/tree/dev/models/wholeBrainSeg_Large_UNEST_segmentation to see what the license condition is for the pre-trained model. Is this model (and all others) licensed as part of the MONAI model zoo package or do separate conditions apply? I guess it may be complicated, since the training data comes from different sources...

Formulated as a feature request: *"as a user I would like to know the license conditions of each model in the zoo"*.</div>

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes #140 .

### Description
This PR adds the description of the license requirements for this repository.
In addition, when solving a CI test error, I also made some changes to address the issue posted in: https://github.com/Project-MONAI/model-zoo/issues/111

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Comment by yiheng-wang-nv** [comment]
Hi @ericspod @wyli @Nic-Ma , could you please help to review the description I added of the license in case my clarification is not clear. Thanks!

**Comment by ericspod** [comment]
Are we requiring that bundles be licensed under the Apache license? There are bundles under other licenses already in the model zoo so we can't change that now, I wouldn't want to restrict people to this license anyhow, I chose MIT for example to be more permissive for what was a rather simple model. In general what we want to say is that one must abide by the license conditions when using the model or deriving anything from it, eg. if you take a bundle and fine tune it for some other task you are required to adhere to the license when distributing that derivative work. These conditions will be discussed in the license files, we should also state that we do not make any guarantee on any bundle's suitability for any task, especially not for diagnostic or therapeutic use.

File: README.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ All source code of models (bundles) are tracked in `models/`, and for each disti
 
 To download and get started with the models, please see [the model browser](https://monai.io/model-zoo.html) and [the example use cases](https://github.com/Project-MONAI/tutorials/tree/main/model_zoo).
 
+## License
+
+The MONAI Model Zoo repository follows [Apache License](https://github.com/Project-MONAI/model-zoo/blob/dev/LICENSE), thus you need to follow this license if using any models (include using pre-trained weights, training scripts and so on). To use the dataset (if existing) that is mentioned in a model, you may also need to follow the corresponding requirements (please check `docs/license.txt` within the bundle directory).
+
```
Comment (ericspod) [inline README.md]: As discussed in my other comment:

```suggestion
## License

Bundles released on the MONAI Model Zoo require a license for the software itself comprising the configuration files and model weights. You are required to adhere to the license conditions included with each bundle, as well as any license conditions stated for data bundles may include or use (please check `docs/license.txt` within the bundle directory). 

The MONAI Model Zoo repository itself follows [Apache License](https://github.com/Project-MONAI/model-zoo/blob/dev/LICENSE), thus you need to follow this license if using its software, and we recommend this license for bundles. 

The MONAI Model Zoo does not make any statement of the suitability of any model for a particular task, especially not for therapeutic or diagnostic use.

```

**Comment by yiheng-wang-nv** [comment]
According to the internal discussions, we have the following requirements about license for bundles in the model zoo:

- There must be a license file naming LICENSE in the root of each bundle, e.g. Apache2.0
- There should be a section “License” in the ReadMe of the bundle
- There should be a data_license.txt file in the doc folder which shows the license of data used in the bundle.

Within my latest commit, I already address these places for bundles that are contributed by internal contributors, and update the readme file and contributing guide.

Hi @ericspod @Nic-Ma @wyli , could you please help to check the updates, thanks!

In addition, here I wish to confirm the license of the following bundles. I did not modify them since I'm not sure about the license.
Hi @ericspod , could you please help to clarify the license of your `mednist_gan`, `valve_landmarks` and `ventricular_short_axis_3label` bundles? Thanks!
Hi @kbressem , could you please help to clarify the license of your `prostate_mri_anatomy` bundle? Thanks!

**Comment by ericspod** [comment]
Hi @yiheng-wang-nv,
* `mednist_gan` was under the same license conditions as MONAI, the data license is what we had for the MedNIST dataset itself which isn't quite a license but it's all we had. 
* `valve_landmarks` and `ventricular_short_axis_3label` are under MIT since I wanted to be permissive for anyone wanting to use these, there is no data license since I can't make that data public. I also have a typo in the `ventricular_short_axis_3label` license file where I didn't replace `Copyright (c) [year] [fullname]` at the top with `Copyright (c) 2022 Eric Kerfoot`.

**Comment by ericspod** [comment]
All the changes look good to me!

File: models/brats_mri_segmentation/docs/README.md [hunk 2]
```diff
@@ -94,3 +94,6 @@ This is an example, not to be used for diagnostic purposes.
 # References
 
 [1] Myronenko, Andriy. "3D MRI brain tumor segmentation using autoencoder regularization." International MICCAI Brainlesion Workshop. Springer, Cham, 2018. https://arxiv.org/abs/1810.11654.
+
+# License
+This model is released under Apache 2.0 terms. The license file is included with the model.
```
Comment (wyli) [inline models/brats_mri_segmentation/docs/README.md]: I think for the Apache license, we should use the recommended way of applying it, for example:
https://github.com/Project-MONAI/model-zoo/blob/71e469723658f1be579423c328a85f7ce97cc52c/models/lung_nodule_ct_detection/LICENSE#L178-L187

Comment (yiheng-wang-nv) [reply models/brats_mri_segmentation/docs/README.md]: Thanks @wyli . I modified the license file. Could you please help to re-review it?

Comment (wyli) [reply models/brats_mri_segmentation/docs/README.md]: sorry I didn't make it clear. I think we should include the full text of the license as file `models/model_name/LICENSE`. Additionally, for the rest of the files in folder `models/model_name` whenever we mention the license we use this paragraph:
https://github.com/Project-MONAI/model-zoo/blob/71e469723658f1be579423c328a85f7ce97cc52c/models/lung_nodule_ct_detection/LICENSE#L189-L201

**Comment by kbressem** [comment]
Thanks! Hi @kbressem , could you please help to clarify the license of your `prostate_mri_anatomy` bundle? Thanks!

Hi @yiheng-wang-nv I've opened a PR to this branch, adding the Apache LICENSE for the prostate model.

File: models/brats_mri_segmentation/docs/README.md (hunk 2 above)
Comment (yiheng-wang-nv) [reply models/brats_mri_segmentation/docs/README.md]: Thanks Wenqi, I updated the LICENSE files and readme.

**Review by wyli — APPROVED** [review]

**Pull request merged**